README.md

```diff
@@ -98,6 +98,7 @@ Read about guest lectures [here](course/guest-lectures.md).
 * [Hsiao-Ching (Jean) Lin](people/hsiaoching-lin.md)
 * [Huai-Che (Acsa) Lu](people/huaiche-lu.md)
 * [Inna Shteinbuk](people/inna-shteinbuk.md)
+* [Jacqueline J.ø Pang](people/jacqueline-pang.md)
```

Comment:
Hi Dan,
Corrected the typo in README file and follow your struction.
Is this Ok now from you monitoring side ?

PS useful guidance for the amend git commend, never use those before!;)

Sent using CloudMagic<https://cloudmagic.com/k/d/mailapp?ct=pi&cv=5.4.47&pv=7.1>

On 周三, 9月 2, 2015 at 14:59, Cornell-CS5356-Fall2015/cs5356 <reply@reply.github.com> wrote:

In README.md<https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/114#discussion_r38570629>:

> @@ -98,6 +98,7 @@ Read about guest lectures [here](course/guest-lectures.md).
>  * [Hsiao-Ching (Jean) Lin](people/hsiaoching-lin.md)
>  * [Huai-Che (Acsa) Lu](people/huaiche-lu.md)
>  * [Inna Shteinbuk](people/inna-shteinbuk.md)
> +* [Jacqueline J.ø Pang](people/jacqueline-pang.md)


You should correct this typo, then amend your last commit via git commit --amend, then force push the change, so git push origin master -f, and it will just update itself here :)

—
Reply to this email directly or view it on GitHub<https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/114/files#r38570629>.